docker-compose.yml

```diff
@@ -1,22 +1,69 @@
+# Starts the Sentinel service and connected monitoring services: Prometheus and Grafana.
+
 version: '3'
+networks:
+  monitoring:
+    driver: bridge
 
 services:
+  # the sentinel image is built from source
   sentinel:
     build: .
     restart: unless-stopped
     env_file: .env
     environment:
       - NODE_ENV=production
       - DB_PATH=data/db.sqlite
+      # hardcode the port inside the container
+      - METRICS_PORT=9100
+    ports:
+      - ${METRICS_PORT:-9100}:9100
+    expose:
+      - 9100
     volumes:
       - data:/app/data
+    networks:
+      - monitoring
     deploy:
       resources:
         limits:
           cpus: '0.50'
           memory: 300M
         reservations:
           memory: 50M
+  prometheus:
+    image: prom/prometheus:v2.36.1
+    container_name: prometheus
+    volumes:
+      - ./prometheus:/etc/prometheus
+      - prometheus_data:/prometheus
+    ports:
+      - ${PROMETHEUS_PORT:-9090}:9090
+    expose:
+      - ${PROMETHEUS_PORT:-9090}
+    networks:
+      - monitoring
+  grafana:
+      image: grafana/grafana:8.2.6
+      container_name: grafana
+      volumes:
+        - grafana_data:/var/lib/grafana
+        - ./grafana/provisioning:/etc/grafana/provisioning
+        - ./grafana/dashboards/:/etc/grafana/dashboards
+        - ./grafana/grafana.ini:/etc/grafana/grafana.ini
+      environment:
+        - GF_AUTH_DISABLE_LOGIN_FORM=true
+        - GF_AUTH_ANONYMOUS_ENABLED=true
+        - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin
```

Comment:
While useful for testing, I don't think we should ship with this config.  
By default it requires login, with credentials preset to admin:admin and a forced pw change on first login. I think that's sane defaults and suggest to remove this env vars before merging.

Reply:
Removed as suggested